<?xml version="1.0" encoding="UTF-8"?>
<launch>

  <node pkg="octomap_navi" type="trajectory_server" name="trajectory_server" output="screen">
    <param name="Vel" value="1.1" />
    <param name="Length" value="1.0" />
    <param name="ld_k" value="1.0" />
    <param name="ld_c" value="0.5" />
    <param name="escape_v" value="0.3" />
  </node>


</launch>

